indexd/index/drivers/alchemy.py

```diff
@@ -720,7 +828,7 @@ def add(
         hashes = hashes or {}
         metadata = metadata or {}
         urls_metadata = urls_metadata or {}
-
+        _enforce_rbac(authz, method="WRITE")
```

Comment:
can we rely on existing authz for write and not overload this feature with that scope? I'd rather keep the record management separate from the external-facing record discovery ability